pkg/manager/manager_tasks.go

```diff
@@ -465,12 +466,18 @@ type totalsGenerate struct {
 func (s *singleton) neededGenerate(scenes []*models.Scene, sprites, previews, markers, transcodes bool) *totalsGenerate {
 
 	var totals totalsGenerate
+	const timeoutSecs = 90 * time.Second
+
+	// create a control channel through which to signal the counting loop when the timeout is reached
 	chTimeout := make(chan struct{})
+
+	//run the timeout function in a separate thread
 	go func() {
-		time.Sleep(90 * time.Second)
+		time.Sleep(timeoutSecs)
 		chTimeout <- struct{}{}
 	}()
 
+	logger.Infof("Counting content to generate.Please wait ...")
```

Comment:
The `Please wait ...` is probably not necessary.

Reply:
I just wanted to make sure the user knows something is going on that might take some time(nothing will be printed before counting is finished or timeout occurs) , maybe just add the ... like this "Counting content to generate ..." ?

Reply:
Yep, `Counting content to generate...` sounds good to me.

Reply:
done